Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -1199,11 +1243,9 @@ class AnimatedStyle extends AnimatedWithChildren {
     for (var key in this._style) {
       var value = this._style[key];
       if (value instanceof Animated) {
```

Comment:
Aren't we going to have a problem if we happen to re-render during an animation? Namely, out-of-date JS values will override up-to-date native values when the view us updated. Contrived example:

Tick 1 (first render):
- js=0
- native=0

Tick 2 (animation driven natively):
- js=0
- native=0.1

Tick 3: (still driving, listener called on the JS side)
- js=0.1
- native=0.2

Tick 4: (still driving, we call a re-render)
- js=0.2
- native=0.3

Tick 5 (re-rendered):
- js=0.2
- native=0.2 (but we'd expect 0.4)

It seems like the old `__getValue`, this updated `__getValue` behavior, and `__getAnimatedValue` all have slightly different intents:

(1) What prop values should we be setting on the native view when we render?
(2) What are the current values of animated props?
(3) What values are we driving and may need update via setNativeProps? (inverse of 1)

It seems like there is a great tension between an animate prop being treated like any other prop, and being managed independently.